dags/inference/configs/jetstream_pytorch_gce_config.py

```diff
@@ -83,22 +83,26 @@ def get_jetstream_pytorch_inference_nightly_config(
       f"export CKPT_PATH={model_configs['checkpoint']}",
       f"export TOKENIZER_PATH=$(pwd)/ckpt_dir/{model_configs['tokenizer']}",
       f"export SHARDING_CONFIG={model_configs['sharding_config']}",
-      "mkdir ckpt_dir",
-      "gsutil cp -r ${CKPT_PATH}/* ckpt_dir/",
+      f"export QUANTIZE={str(model_configs['quantize'])}",
+      "mkdir /dev/shm/ckpt_dir",
+      "gsutil cp -r ${CKPT_PATH}/* /dev/shm/ckpt_dir/",
       # Start jetstream-pytorch server in the background
       """python run_server.py \
         --model_name=${MODEL_NAME} \
         --size=${SIZE} \
         --batch_size=${BATCH_SIZE} \
         --max_cache_length=${MAX_CACHE_LEN} \
-        --checkpoint_path=ckpt_dir \
+        --checkpoint_path=/dev/shm/ckpt_dir \
         --tokenizer_path=${TOKENIZER_PATH} \
+        --quantize_weights=${QUANTIZE} \
+        --quantize_kv_cache=${QUANTIZE} \
```

Comment:
Since there may be cases where we don't quantize kv-cache, can we have a separate flag for these? i.e. 
```
        --quantize_weights=${QUANTIZE} \
        --quantize_kv_cache=${QUANTIZE_KV_CACHE} \
```

Reply:
Currently there's no such case in jetstream-pytorch, but I will add it